.github/workflows/create-release-pr.yaml

```diff
@@ -11,10 +11,35 @@ jobs:
     name: Create Release PR
     runs-on: ubuntu-latest
     steps:
+      - name: Check if a branch is whitelisted and maintained
+        id: branch-check
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          WHITELISTED_BRANCHES=("main" "release/*")
+
+          IS_WHITELISTED=false
+          # Check the branch against the whitelist using wildcard matching
+          for BRANCH in "${WHITELISTED_BRANCHES[@]}"; do
```

Comment:
can't you just do instead of checking manually? Not well documented, but seems like it might work?
```
on:
  workflow_dispatch:
    branches:
      - main
      - 'releases/*'
```